README.md

```diff
@@ -124,35 +124,37 @@ After installation the reference to these resources has to be added through menu
 
 This is not complicated, but you'll need to work precise. The configuration of HA is stored in .yaml files, these can be edited in the HA file editor (in the left main menu).
 
-  > If you have installed V2G Liberty before, please remove any changes made during that installation to the .yaml files.
+  > For version 0.2.0 the folder structure has changed significantly due to AppDaemon requirements, please check carefully.
```

Comment:
Maybe add extra hints informing the migrating user that:
- Most files should have been moved automatically when upgrading
- One way to check is see if the files are in the right place: as shown below, there should be two `v2g-liberty` folders now, one under `packages` and the other under `apps`.
- The `secrets.yaml` should also have been moved automatically, I guess.

Reply:
1. True, i'll clarify that in the e-mail that i'll send out.
2. Same
3. No, that stays in the same location.